src/Algebra/Graph/Relation/InternalDerived.hs

```diff
@@ -66,27 +59,146 @@ The 'Show' instance produces symmetrically closed expressions:
 
 @show (1     :: SymmetricRelation Int) == "vertex 1"
 show (1 * 2 :: SymmetricRelation Int) == "edges [(1,2),(2,1)]"@
+
+The 'Graph' instance respects the comutativity of connect.
+
+The 'Num' instance respects the comutativity of connect.
+
+The total order on graphs is defined using /size-lexicographic/ comparison:
+
+* Compare the number of vertices. In case of a tie, continue.
+* Compare the sets of vertices. In case of a tie, continue.
+* Compare the number of edges. In case of a tie, continue.
+* Compare the sets of edges.
+
+Here are a few examples:
+
+@'vertex' 1 < 'vertex' 2
+'vertex' 3 < 'Algebra.Graph.AdjacencyMap.edge' 1 2
+'vertex' 1 < 'Algebra.Graph.AdjacencyMap.edge' 1 1
+'Algebra.Graph.AdjacencyMap.edge' 1 1 < 'Algebra.Graph.AdjacencyMap.edge' 1 2
+'Algebra.Graph.AdjacencyMap.edge' 1 2 > 'Algebra.Graph.AdjacencyMap.edge' 1 1 + 'Algebra.Graph.AdjacencyMap.edge' 2 2
+'Algebra.Graph.AdjacencyMap.edge' 1 2 < 'Algebra.Graph.AdjacencyMap.edge' 1 3@
```

Comment:
It may be useful to add comparison between `edge 1 2` and `edge 2 1` into this example. They should compare as equal.

Reply:
Makes sense! Added!